tensorflow/python/kernel_tests/pooling_ops_test.py

```diff
@@ -839,9 +839,14 @@ def testMaxPoolingWithArgmax(self):
         1.0, 1.0, 1.0, 1.0, 0.0, 1.0, 1.0, 1.0, 1.0, 1.0, 0.0, 1.0, 0.0, 0.0,
         0.0, 1.0, 0.0, 1.0
     ]
-    configs = [[False, False, [0, 1, 3, 5, 0, 2, 6, 8]],
-               [False, True, [0, 1, 3, 5, 9, 11, 15, 17]],
-               [True, False, [0, 1, 3, 5, 0, 2, 6, 8]]]
+    configs = [
```

Comment:
I think these configs should be namedtuples; this way they'll be self-documenting

Reply:
Great idea!